code/pizzabot.js

```diff
@@ -1,7 +1,98 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+// 1. Definition of variables 
 
+const vegetarian = "Vegetarian"
+const hawaiian = "Hawaiian"
+const pepperoni = "Pepperoni"
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// 2. Functions
+
+// Function for validating ordername
+const validateOrderName = (orderName) => {
+    if (orderName.toLowerCase() === vegetarian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === hawaiian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === pepperoni.toLowerCase()) {
+        return true;
+    } else {
+        alert('Select a pizza from the menu!');
+        return false;
+    }
+};
+
+// Function for calculating total cost
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    let orderTotal = orderQuantity * pizzaPrice;
+    return orderTotal;
```

Comment:
We can simplify this function. Since you are creating `let` variable (should be `const` btw because you are not overwriting its value anywhere) and returning it very next line, we could declare it like this 
```
const calculateTotalCost = (orderQuantity, pizzaPrice) => {
    return orderQuantity * pizzaPrice;
}
```
Another tip from my side - since we have only one line inside `calculateTotalCost` function, we can write it in "arrow function shorthand" :

`const calculateTotalCost = (orderQuantity, pizzaPrice) => orderQty * pizzaPrice;`

We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside. At the end, this snippet of code is exactly the same as the first one I wrote above.

For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp